### PR TITLE
(cheevos) show RetroAchievements Hash in content information list

### PR DESCRIPTION
**Pull request by Jamiras**
## Description

Adds "RetroAchievements Hash" as a line item in the "Content Quick Menu > Information" submenu:

image

This provides a way for the user to identify which version of a game they're using when reporting an issue with an achievement. 

If the hash could not be resolved, "N/A" will be displayed. This is a limitation of the current identification process as more than one hash may be tried, and an incompatible hash provides very little information other than the fact that it's incompatible. Perhaps a different label (like "No Match") would be preferred?

For similar reasons, the line is not shown at all when viewing "Information" without loading the content (i.e. from a playlist). As we have not tried to resolve the hash, we can't display the result.

NOTE: this also moves the "Database" line down past the "Play Time" lines so it's next to the "Database Entry" item. The new order is "Name", "File Path", "Core", "Play Time", "Last Played", "RetroAchievements Hash", "Database", "Database Entry". "Database" used to be between "Core" and "Play Time".

## Related Issues

n/a

## Related Pull Requests

n/a

## Reviewers

@leiradel @meleu

### CODE REVIEW

File: msg_hash.h [hunk 1]
```diff
@@ -1761,6 +1761,7 @@ enum msg_hash_enums
    MENU_LABEL(CONTENT_INFO_DATABASE),
    MENU_LABEL(CONTENT_INFO_RUNTIME),
    MENU_LABEL(CONTENT_INFO_LAST_PLAYED),
+   MENU_LABEL(CONTENT_INFO_RETROACHIEVEMENTS_HASH),
```
Comment (meleu) [inline msg_hash.h]: maybe `CONTENT_INFO_CHEEVOS_HASH` for consistency sake?

Comment (Jamiras) [reply msg_hash.h]: There isn't really any cohesive consistency between the existing labels. _Most_ are CHEEVOS, but there's a couple that use "RETRO_ACHIEVEMENTS". So I'm not really consistent with either. Will change to CHEEVOS.

**Comment by i30817** [comment]
This is probably a bad idea on this PR but could a button be added in the info page to copy all of the info to the clipboard?

I feel like asking people to write large checksum strings is more than a little unusable.

**Comment by Jamiras** [comment]
Any clipboard implementation would have to be implemented to work on all supported systems, and I didn't see any references to "clipboard" anywhere in the existing codebase.

Additionally, some users will be playing on systems separate from where they would be reporting issues (i.e. raspberry pi installations). Even if they don't transcribe it correctly, the number of hashes associated to any specific game is pretty limited, so the part provided should be sufficient to determine the correct value.

**Pull request merged**